index.html

```diff
@@ -24,15 +24,15 @@
         <div class="logo">
             <a href="#">My Logo</a>
         </div>
-        <ul class="navigation-menu">
+        <ul class="navigation-menu" id="nm">
             <li><a class="navigation" href="#works">Portfolio</a></li>
             <li><a class="navigation" href="#aboutme">About</a></li>
             <li><a class="navigation" href="#contact">Contact</a></li>
         </ul>
         <div class="hamburger">
```

Comment:
- [ ] The menu should be displayed all the way to the bottom and users should not be able to click other buttons behind the menu when menu is open
![image](https://user-images.githubusercontent.com/62705947/217586294-e865d752-833f-4448-b9df-3dead9e19dae.png)

Reply:
- [ ] Adjust the items and the X as on the design

Yours:
![image](https://user-images.githubusercontent.com/62705947/217586634-8d713182-2e4a-470c-afa4-28f8048dd63b.png)
Design:
![image](https://user-images.githubusercontent.com/62705947/217586691-47e17668-d5c7-4c75-8650-867976be5963.png)

Reply:
On my end the menu overlay is displayed all the way down the page, can not really understand why it is not on your end.